src/d_player.h

```diff
@@ -250,6 +250,7 @@ typedef enum
 	k_lapanimation,		// Used to show the lap start wing logo animation
 	k_laphand,			// Lap hand gfx to use; 0 = none, 1 = :ok_hand:, 2 = :thumbs_up:, 3 = :thumps_down:
 	k_cardanimation,	// Used to determine the position of some full-screen Battle Mode graphics
+	k_oldcardanimation,	// Used to interpolate full-screen Battle Mode graphics
```

Comment:
This can probably cause desynch with vanilla clients since kartstuff is synched so fields after k_oldcardanimations will be synched wrong

Reply:
This breaks netcompability with vanilla hard. Joining a server hosted with this results in the game throwing an I_Error 
![Screenshot (136)](https://github.com/Indev450/SRB2Kart-Saturn/assets/136046486/162025d9-476c-461c-8f35-60d2ff8091ff)

Reply:
Shouldn't have rushed that one...
Turns out there *was* a better way after all!